run.sh

```diff
@@ -98,6 +103,11 @@ if [ -z "$GALAXY_CONFIG_FILE" ]; then
     export GALAXY_CONFIG_FILE
 fi
 
+if [ $INITIALIZE_TOOL_DEPENDENCIES -eq 1 ]; then
+    # Install Conda environment if needed.
+    python ./scripts/manage_tool_dependencies.py -c "$GALAXY_CONFIG_FILE" init_if_needed
```

Comment:
@jmchilton @natefoo Should this be run as part of ` ./scripts/common_startup.sh` (called at line 66) instead of only when Galaxy is started through `run.sh` ?